README.md

```diff
@@ -1,92 +1,73 @@
+   ```javascript
+   import { Fluence, randomKras } from "@fluencelabs/js-client";
 
    Fluence.connect(randomKras());
    ```
 
-### Node.js Apps
+### HTML page
 
-**Prerequisites:**
+Add a script tag with the JS Client bundle to your `index.html`. The easiest way to do this is using a CDN (
+like [JSDELIVR](https://www.jsdelivr.com/) or [UNPKG](https://unpkg.com/)).
 
-The Fluence JS Client only supports the ESM format. This implies that a few preliminary steps are required if your project is not already using ESM:
+Here is an example using the JSDELIVR CDN:
 
-- Add `"type": "module"` to your package.json.
-- Replace `"main": "index.js"` with `"exports": "./index.js"` in your package.json.
-- Remove `'use strict';` from all JavaScript files.
-- Replace all `require()`/`module.export` with `import`/`export`.
-- Use only full relative file paths for imports: `import x from '.';` → `import x from './index.js';`.
-
-If you are using TypeScript:
-
-- Make sure you are using TypeScript 4.7 or later.
-- Add [`"module": "ESNext", "target": "ESNext", "moduleResolution": "nodenext"`](https://www.typescriptlang.org/tsconfig#module) to your tsconfig.json.
-- Use only full relative file paths for imports: `import x from '.';` → `import x from './index.js';`.
-- Remove `namespace` usage and use `export` instead.
-- You must use a `.js` extension in relative imports even though you're importing `.ts` files.
-
-**Installation:**
+   ```html
+   <head>
+     <title>Cool App</title>
+     <script
+       src="https://cdn.jsdelivr.net/npm/@fluencelabs/js-client/dist/browser/index.js"
+     ></script>
+   </head>
+   ```
 
-1. Install the following packages:
+If you cannot or don't want to use a CDN, feel free to get the script directly from
+the [npm package](https://www.npmjs.com/package/@fluencelabs/js-client) and host it yourself. You can find the script in
+the `/dist` directory of the package. (Note: this option means that developers understand what they are doing and know
+how to serve this file from their own web server.)
 
-   ```
-   npm i @fluencelabs/js-client.api"@fluencelabs/js-client.node @fluencelabs/fluence-network-environment
-   ```
+After importing JS-client to HTML page the client is available as ```window.Fluence``` variable.
+To get a specific network you can peek at
 
-2. Add the following lines at the beginning of your code:
+```
+https://cdn.jsdelivr.net/npm/@fluencelabs/js-client/dist/network.js
+```
 
-   ```
-   import '@fluencelabs/js-client.node';
-   import { Fluence } from "@fluencelabs/js-client.api";
-   import { randomKras } from '@fluencelabs/fluence-network-environment';
+and hardcode selected network. So initialization would look like this
 
-   Fluence.connect(randomKras());
-   ```
+```javascript
+    // Passing 1 kras network config from ./dist/network.js above
+    window.Fluence.connect({
+        multiaddr: "/dns4/0-kras.fluence.dev/tcp/9000/wss/p2p/12D3KooWSD5PToNiLQwKDXsu8JSysCwUt8BVUJEqCHcDe7P5h45e",
```

Comment:
In the future, addresses may be changed. And as I discovered @nahsi used to store all currently implemented addresses in the repo file: https://github.com/fluencelabs/fluence-network-environment/blob/main/src/index.ts

Maybe we should link the source info of @nahsi in any places we can?

ps. In other places where the same comment should be written, I will not write the one, but I want to consider the same comment